cmake/OpenCVFindFrameworks.cmake

```diff
@@ -32,6 +32,15 @@ if(WITH_OPENMP)
   if(OPENMP_FOUND)
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
+    # NOTES:
+    # 1. For CMake <= 3.5.1, OpenMP_CXX_INCLUDE_DIRS is not defined.
```

Comment:
> 3.5.1

This is not correct.
This is a minimal CMake version on which we take a look look for OpenCV 4.x.

https://cmake.org/cmake/help/v3.6/module/FindOpenMP.html

- still no `_INCLUDE_DIRS` too.

Reply:
I checked CMake docs and found `_INCLUDE_DIRS` is defined since CMake>=3.16.0. See https://cmake.org/cmake/help/v3.16/module/FindOpenMP.html#backward-compatibility.